pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,322 @@ excerpt:
+Required if ca_certificate_path is specified and if the custom CA certificate is an *intermediate CA certificate*. 
+
+This needs to contain all CA certificates comprising the complete sequence starting precisely with the CA certificate that was used to sign the custom CA certificate and ending with a root CA certificate (where issuer and subject are equivalent), yielding a gapless certification path. The order is significant and the list has to contain at least one certificate.
+
+
+# Installing DC/OS Enterprise with a custom CA certificate
+## Starting point
+Based on the requirements described above, this is the starting point for the installation:
+
+- The installation of DC/OS Enterprise via the Advanced Installer has been prepared according to the corresponding [documentation](/1.10/installing/ent/custom/advanced/).
+
+- On the bootstrap node, the files carrying custom CA certificate, the associated private key and, optionally, the CA certificate chain have been placed into the `$DCOS_INSTALL_DIR/genconf/` directory (see the [section](#glossary-of-the-terms-used-in-this-documentation) above for more detailed description), Example (commands executed on the bootstrap node):
+
+```bash
+cd $DCOS_INSTALL_DIR
+ls genconf/
+```
+```bash 
+dcos-ca-certificate.crt
+dcos-ca-certificate-key.key
+dcos-ca-certificate-chain.crt
+```
+
+- The custom CA private key has been securely placed on all DC/OS master nodes and is sufficiently protected (refer to [this section](#manually-placing-the-private-key-associated-with-the-custom-ca-certificate-onto-the-master-nodes) for more details). Example (command issued on one of the DC/OS master nodes):
+
+```bash
+stat /var/lib/dcos/pki/tls/CA/private/custom_ca.key
+```
+```bash
+File: ‘/var/lib/dcos/pki/tls/CA/private/custom_ca.key’
+Size: 9             Blocks: 8          IO Block: 4096   regular file
+Device: ca01h/51713d    Inode: 100671105   Links: 1
+Access: (0600/-rw-------)  Uid: (    0/    root)   Gid: (    0/    root)
+Context: unconfined_u:object_r:var_lib_t:s0
+Access: 2017-12-27 12:35:58.643278377 +0000
+Modify: 2017-12-27 12:35:58.643278377 +0000
+Change: 2017-12-27 12:36:13.019162417 +0000
+Birth: -
+```
+
+- The configuration parameters `ca_certificate_path`, `ca_certificate_key_path` and `ca_certificate_chain_path` are specified in the DC/OS configuration file `$DCOS_INSTALL_DIR/genconf/config.yaml` and point to the relevant locations in the file system. Example (commands issued on the bootstrap node):
+
+```bash
+cd $DCOS_INSTALL_DIR
+cat genconf/config.yaml
+```
+```bash
+[...]
+ca_certificate_path: genconf/dcos-ca-certificate.crt
+ca_certificate_key_path: genconf/dcos-ca-certificate-key.key
+ca_certificate_chain_path: genconf/dcos-ca-certificate-chain.crt
+[...]
+```
+Note that `ca_certificate_chain_path` is an optional parameter when setting up DC/OS Enterprise with a custom CA certificate when the latter is a self-signed root certificate.
```

Comment:
Describing this as an "optional parameter" suggests that it is not important whether it is present or not. However, other sections suggest that it should definitely not be present in this case. Make it more direct, e.g.:

Note that `ca_certificate_chain_path` must not be present when setting up DC/OS Enterprise with a self-signed root certificate as the custom CA certificate.